versions/1/kibana/spec.yml

```diff
@@ -9,7 +9,7 @@
       - description: A dashboard asset file
         type: file
         contentMediaType: "application/json"
-        pattern: '.+\.json'
+        pattern: '{PACKAGE_NAME}+-.+\.json'
```

Comment:
This is an interesting idea - to allow certain keywords in the spec that can act as variables/placeholders whose value depends on the context. TBH, this is not how I had imagined implementing semantic validations but I like it - its simple! Let's go with this idea and see how far we can take it.

We should probably document these keywords somewhere (outside the spec, of course) for spec developers so they know which keywords are available to them and what they mean.

Reply:
Do you think about README or CONTRIBUTING file in package-spec or somewhere else?

Reply:
I think CONTRIBUTING is the right place for this sort of thing. Let's try to keep the README more for spec _consumers_ and CONTRIBUTING for spec _contributors_, although there might be some overlap/fuzziness, which is okay IMHO.

Reply:
I added a short one.